webpack/plugins/i18n.js

```diff
@@ -0,0 +1,50 @@
+// i18n webpack generator plugin
+// inspired by https://github.com/webpack-contrib/copy-webpack-plugin/blob/master/src/index.js
+const cpx = require('cpx')
+const manageTranslations = require('react-intl-translations-manager').default;
+
+function i18nGenerator (src, dest) {
+  const afterEmit = async (compilation, cb) => {
+    try {
+      await doTranslate(cb);
+      cpx.copy(src, dest)
+    } catch (error) {
+      console.error('an error occured >>', error)
+    } finally {
+      cb();
+    }
+  }
+
+  const doTranslate = () => {
+    return new Promise(function(resolve, reject) {
+      manageTranslations({
+        messagesDirectory: './dist-build/messages/',
+        translationsDirectory: './dist-build/locales/',
+        languages: ['en', 'de']
```

Comment:
how about extracting the value (`['en', 'de']`) to a `const` and move it to the top, so it's more exposed and easier to find once a new language is supported?